### ISSUE TITLE
Strange Code Comments

### ISSUE THREAD

**Issue by fakabbir** [pdfminer/pdfminer.six#282]
` font = PDFType1Font(self, spec)  # this is so wrong!`
I found this in the code. IT there something really wrong with this line ?

https://github.com/pdfminer/pdfminer.six/blob/master/pdfminer/pdfinterp.py#L216

**Comment by pietermarsman** [comment]
It is probably so wrong because the font subtype is something unexpected or unkown and the `PDFType1Font` is used as a default. Not sure if this clause is ever triggered.

In general, I guess that pdfminer tries to be robust and parses all different kinds of wrong pdfs as good as possible.

**Comment by pietermarsman** [comment]
This isn't the only strange comment though. A lot of comments also specify the name of the class that is defined after the comment.

Could you make a list of "strange comment types" that we should remove?

**Comment by fakabbir** [comment]
Sure, Let me grab some patience and time.

**Comment by jstockwin** [comment]
@fakabbir I appreciate it's been a while, but I'm going through all of the open issues. Are you still hoping to get around to this at some point?

**Comment by fakabbir** [comment]
@jstockwin Yes, probably I would devote some time for this and some other issues too.

**Comment by jstockwin** [comment]
That's great, thanks!
